crates/bevy_render/src/lib.rs

```diff
@@ -137,6 +137,7 @@ impl Plugin for RenderPlugin {
         app.add_asset::<Shader>()
             .add_debug_asset::<Shader>()
             .init_asset_loader::<ShaderLoader>()
+            .init_resource::<view::screenshot::ScreenshotManager>()
```

Comment:
Should this be behind a feature flag? I assume it's not a big deal, but it doesn't really need to be present all the time either.

Reply:
Actually, I'm curious if the entire feature could be behind a flag? I don't want to block on that, I'm just curious if it's possible/if it's even worth it.

Reply:
Seems generally useful for rendering testing as it would allow for golden image testing, and it seems fairly trivial for a user to make a player facing screenshot keyboard shortcut too. Unless I'm seeing a hidden cost here. seems like a general win to me.

Reply:
Yeah, I was more thinking about not needing to compile it if it's unused, but this is probably too small to be worth the complexity of a feature flag.

Reply:
I didn't think about a player screenshot feature though, good idea.

Reply:
A cost is the new dependency on `bevy_task`.

Reply:
bevy_tasks was already a dependency through bevy_ecs, this doesn't really change anything.

Reply:
Yeah I don't feel this needs to be feature flagged.